lib/rules/valid-v-on.js

```diff
@@ -149,12 +150,26 @@ module.exports = {
             })
           }
         }
-        if (!utils.hasAttributeValue(node) && !node.key.modifiers.some(VERB_MODIFIERS.has, VERB_MODIFIERS)) {
-          context.report({
-            node,
-            loc: node.loc,
-            message: "'v-on' directives require that attribute value or verb modifiers."
-          })
+
+        if (
+          !utils.hasAttributeValue(node) &&
+          !node.key.modifiers.some(VERB_MODIFIERS.has, VERB_MODIFIERS)
+        ) {
+          if (node.value && sourceCode.getText(node.value.expression)) {
+            const value = sourceCode.getText(node.value)
+            context.report({
+              node,
+              loc: node.loc,
+              message: 'Avoid using JavaScript keyword as "v-on" value: {{value}}.',
+              data: { value }
+            })
+          } else {
+            context.report({
+              node,
+              loc: node.loc,
+              message: "'v-on' directives require a value or verb modifiers (like 'stop' or 'prevent')."
```

Comment:
we should not use plural word here, besides of that it looks fine 👍  
```suggestion
              message: "'v-on' directives require a value or verb modifier (like 'stop' or 'prevent')."
```